src/styles/components/_menu.scss

```diff
@@ -28,6 +28,7 @@
   background: map-get($roles, menu);
   height:100%;
   color: $charcoal;
+  overflow-y: auto;
```

Comment:
Lets do this, if you have time. We need a visual affordance to indicate something is scrollable, and I would like it to be consistent.